### DISCUSSION TITLE
[Feature] Automatically remove duplicates during mobile backup

### DISCUSSION THREAD

**Discussion by xiaowuap** [immich-app/immich#14396] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I previously used Synology Photos, which automatically skips files with the same name during mobile uploads. Now, I'm migrating to TrueNAS with Immich. It seems that the Immich mobile client cannot automatically deduplicate my existing photos, which I've mounted into Immich via an external folder.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by mmomjian** [comment]
Duplicate: #9899
